src/file/tokio_compat_file.rs

```diff
@@ -303,7 +319,10 @@ impl TokioCompatFile {
         self.project().inner
     }
 
-    fn flush_one(self: Pin<&mut Self>, cx: &mut Context<'_>) -> Poll<Result<(), std::io::Error>> {
+    fn flush_pending_requests(
+        self: Pin<&mut Self>,
+        cx: &mut Context<'_>,
+    ) -> Poll<Result<(), std::io::Error>> {
```

Comment:
This function doesn't await on anything:

```suggestion
    ) -> Result<(), std::io::Error> {
```